src/common/components/header/index.js

```diff
@@ -2,74 +2,106 @@ import React, { Component, PropTypes } from 'react';
 
 import { signOut } from '../../actions/auth-store';
 
-import containerStyles from '../../containers/container.css';
+import { conf } from '../../helpers/config';
+
 import style from '../../style/vanilla/css/navigation.css';
+import { IconChevron, IconUser } from '../vanilla-modules/icons';
 
 const brandmark = 'https://assets.ubuntu.com/v1/7f93bb62-snapcraft-logo--web-white-text.svg';
 
 export default class Header extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showUserDropdown: false
+    };
+  }
+
   render() {
     const { authenticated, user } = this.props;
 
+    const SNAPCRAFT_URL = conf.get('SNAPCRAFT_URL');
```

Comment:
This could be moved to top of the module, no need to read it from conf on every render.

Reply:
done